src/cli/Cli.spec.ts

```diff
@@ -104,6 +104,62 @@ class MockCommandWithValidation extends AnonymousCommand {
   }
 }
 
+class MockCommandWithBooleanCuration extends AnonymousCommand {
+  public get name(): string {
+    return 'cli mock boolean curation';
+  }
+  public get description(): string {
+    return 'Mock command with boolean curation';
+  }
+  constructor() {
+    super();
+
+    this.options.push(
+      {
+        option: '-a, --parameterA [parameterA]'
+      },
+      {
+        option: '-b, --parameterB [parameterB]'
+      },
+      {
+        option: '-c, --parameterC [parameterC]'
+      },
+      {
+        option: '-d, --parameterD [parameterD]'
+      },
+      {
+        option: '-e, --parameterE [parameterE]'
+      },
+      {
+        option: '-f, --parameterF [parameterF]'
+      },
+      {
+        option: '-g, --parameterG [parameterG]'
+      },
+      {
+        option: '-h, --parameterH [parameterH]'
+      },
+      {
+        option: '-x, --parameterH [parameterX]'
+      }
+    );
+
+    this.types.boolean.push('a', 'parameterA');
+    this.types.boolean.push('b', 'parameterB');
+    this.types.boolean.push('c', 'parameterC');
+    this.types.boolean.push('d', 'parameterD');
+    this.types.boolean.push('e', 'parameterE');
+    this.types.boolean.push('f', 'parameterF');
+    this.types.boolean.push('g', 'parameterG');
+    this.types.boolean.push('h', 'parameterH');
+    this.types.boolean.push('x', 'parameterX');
```

Comment:
We can simplify this code block and push all of them at once. For readability, we can put the pairs on separate lines, but it's unnecessary to call the push method so many times.